addons/project/static/src/project_sharing/chatter/suggestion_service_patch.js

```diff
@@ -13,9 +13,8 @@ patch(SuggestionService.prototype, {
                 { abortSignal }
             );
             this.store.insert(suggestedPartners);
-            const suggestedPartnersIds = suggestedPartners["res.partner"].map(
-                (partner) => partner.id
-            );
+            const suggestedPartnersIds =
+                suggestedPartners["res.partner"]?.map((partner) => partner.id) ?? [];
             thread.limitedMentions = Object.values(this.store.Persona.records).filter((persona) =>
                 suggestedPartnersIds.includes(persona.id)
             );
```

Comment:
I would suggest:
```suggestion
            thread.limitedMentions = suggestedPartnersIds.map((id) => ({ id, type: "partner" }));
```
So we avoid iterating on the personas in the case of empty `suggestedPartnersIds` and also insert the data directly from the server side.